modules/nodes-pods-autoscaling-custom-trigger-auth.adoc

```diff
@@ -0,0 +1,127 @@
+// Module included in the following assemblies:
+//
+// * nodes/nodes-pods-autoscaling-custom.adoc
+
+:_content-type: CONCEPT
+[id="nodes-pods-autoscaling-custom-trigger-auth_{context}"]
+= Understanding custom metrics autoscaler trigger authentications
+
+A trigger authentication allows you to include authentication information in a scaled object or a scaled job that can be used by the associated containers. You can use trigger authentications to pass {product-title} secrets, platform-native pod authentication mechanisms, environment variables, and so on.   
+
+Some triggers, or scalers, allow you to set specific authentication parameters by using a trigger authentication. For information, see each trigger definition in the link:https://keda.sh/docs/2.7/scalers/[KEDA Scalers] documentation.
```

Comment:
Is Red Hat supporting all of the Scalers listed by the upstream? IE: if a customer finds a bug with one of these scalers (or the upstream adds a scaler in the middle of a 4.Y version's lifecycle) can a customer take advantage of it? Can they get patches and security updates for the scalers?

If the answer is no; to this then we shouldn't like to the upstream here; but instead we should list what Red Hat is supporting.

All in all I don't have an issue linking to the upstream here; but we have to clarify this paragraph with a support clause / call out that said (more or less): Red Hat provides and only supports the ability to use scalers; the source / support for each individual scaler is offered by the provider of at scaler (please see the provider for support options), etc, etc.